contributions/catalog/webapps/templates/contribute/capability_details.html

```diff
@@ -30,156 +26,175 @@
-                                                    <td>{{item[1]}}</td>
-                                                {% endif %}
-                                            {% endfor %}
-                                        </tr>
-                                    {% endfor %}
-                                </table>
-                            {% endfor %}
-                        </td>
-                    </tr>
-                    {% endif %}
-                    {% if post.deploymentDetails.location %}
-                    <tr>
-                        <td><b>Location&nbsp;&nbsp;</b></td>
-                        <td>{{post.deploymentDetails.location}}</td>
-                    </tr>
-                    {% endif %}
-                </table>
-            </p>
-        </div>
+            {% endif %}
+            {% if post.healthCheckUrl %}
+            <div class="row">
+                <div class="col-12 col-sm-3">
+                    <h4>Health Check URL </h4>
+                </div>
+                <div class="col-12 col-md-9">
+                    <span class="small-body-text">{{post.healthCheckUrl}}</span>
+                </div>
+            </div>
+            {% endif %}
+            {% if post.dataDeletionEndpointDetails %}
+            <div class="row">
+                <div class="col-12 col-sm-3">
+                    <h4>Data Deletion Endpoint Details </h4>
+                </div>
+                <div class="col-12 col-md-9">
+                    <table>
+                        {% if post.dataDeletionEndpointDetails.apiKey %}
+                        <tr id="datadeletion_row">
+                            <td><b>API Key&nbsp;&nbsp;</b></td>
+                            <td class="hidetext">{{post.dataDeletionEndpointDetails.apiKey}}</td>
+                            <td> &#160 &#160 <i class="bi bi-eye-slash field_icon" id="toggleText" style="cursor: pointer;"></i></td>
+                        </tr>
+                        {% endif %}
+                        {% if post.dataDeletionEndpointDetails.deletionEndpoint %}
+                        <tr>
+                            <td><b>Deletion Endpoint&nbsp;&nbsp;</b></td>
+                            <td>{{post.dataDeletionEndpointDetails.deletionEndpoint}}</td>
+                        </tr>
+                        {% endif %}
+                        {% if post.dataDeletionEndpointDetails.description %}
+                        <tr>
+                            <td><b>Description&nbsp;&nbsp;</b></td>
+                            <td>{{post.dataDeletionEndpointDetails.description}}</td>
+                        </tr>
+                        {% endif %}
+                    </table>
+                </div>
+            </div>
+            {% endif %}
+            {% if post.deploymentDetails %}
+            <div class="row">
+                <div class="col-12 col-sm-3">
```

Comment:
```suggestion
                <div class="col-12 col-md-3">
```

There are other places where col-sm-* probably needs to be changed to col-md-*. Thanks.

Reply:
updated